modules/dnn/src/tokenizer/unicode-data.hpp

```diff
@@ -0,0 +1,18 @@
+#include <cstdint>
```

Comment:
Need to add opencv license:

```
// This file is part of OpenCV project.
// It is subject to the license terms in the LICENSE file found in the top-level directory
// of this distribution and at http://opencv.org/license.html.
```

And the original source and license of this file.